src/tile/Tile.js

```diff
@@ -122,7 +122,7 @@ const Tile = props => {
 Tile.propTypes = {
   title: PropTypes.string,
   icon: PropTypes.object,
-  caption: PropTypes.string,
+  caption: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
This will still throw an error if you pass a component to it. Look at PropTypes.element or PropTypes.node. https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
I tried it in react-native-element-app and it worked quite well without any errors. Also when I passed simple `<Text>Test</Text>` the `typeof` it was still an `object`

Reply:
Did you use a Text component from react-native or the native one?

Reply:
I tried Text from react-native and react-native-elements
Also Tried with 
```
<View>
  <Avatar
    size="large"
    source={{uri: "https://s3.amazonaws.com/uifaces/faces/twitter/brynn/128.jpg"}}
    onPress={() => console.log("Works!")}
    activeOpacity={0.7}
  />
  <Text>Test</Text>
</View>
```
and still `typeof caption` is equal `string`

Reply:
Maybe we can change it to `PropTypes.node`
```
Anything that can be rendered: numbers, strings, elements or an array
(or fragment) containing these types.
```
and it will accept everything (strings and components) needed without any problems :)

Reply:
Alright then